README.md

```diff
@@ -1,2 +1,42 @@
-# Project-1-Mobile
- 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project 1-Mobile .
+
```

Comment:
These are optional. But I recommended. 
- [ ] You can write a few  sentence about  your project.
- [ ] Also you can add screenshot of your project.